Crash: onDestroy - CardBrowser.invalidate()

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performDestroyActivity (ActivityThread.java:5176)
  at android.app.ActivityThread.handleDestroyActivity (ActivityThread.java:5195)
  at android.app.servertransaction.DestroyActivityItem.execute (DestroyActivityItem.java:39)
  at android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:145)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:70)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2199)
  at android.os.Handler.dispatchMessage (Handler.java:112)
  at android.os.Looper.loop (Looper.java:216)
  at android.app.ActivityThread.main (ActivityThread.java:7625)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:524)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:987)
Caused by: java.lang.NullPointerException: 
  at com.ichi2.anki.CardBrowser.invalidate (CardBrowser.java:1275)
  at com.ichi2.anki.CardBrowser.onDestroy (CardBrowser.java:713)
  at android.app.Activity.performDestroy (Activity.java:7737)
  at android.app.Instrumentation.callActivityOnDestroy (Instrumentation.java:1327)
  at android.app.ActivityThread.performDestroyActivity (ActivityThread.java:5161)
  at android.app.ActivityThread.handleDestroyActivity (ActivityThread.java:5195)
  at android.app.servertransaction.DestroyActivityItem.execute (DestroyActivityItem.java:39)
  at android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:145)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:70)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2199)
  at android.os.Handler.dispatchMessage (Handler.java:112)
  at android.os.Looper.loop (Looper.java:216)
  at android.app.ActivityThread.main (ActivityThread.java:7625)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:524)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:987)
```

--- a/AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.java
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.java
@@ -136,7 +136,8 @@ public class CardBrowser extends NavigationDrawerActivity implements
 
     /** List of cards in the browser.
     * When the list is changed, the position member of its elements should get changed.*/
-    private List<CardCache> mCards;
+    @NonNull
+    private List<CardCache> mCards = new ArrayList<>();
     private ArrayList<Deck> mDropDownDecks;
     private ListView mCardsListView;
     private SearchView mSearchView;

